osm_fieldwork/CSVDump.py

```diff
@@ -322,14 +322,21 @@ def main():
 
     # if verbose, dump to the terminal.
     if args.verbose is not None:
-        root = logging.getLogger()
-        root.setLevel(logging.DEBUG)
-
-        ch = logging.StreamHandler(sys.stdout)
-        ch.setLevel(logging.DEBUG)
-        formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
-        ch.setFormatter(formatter)
-        root.addHandler(ch)
+        logging.basicConfig(
+            level=logging.DEBUG,
+            format=("%(asctime)s - %(name)s - %(levelname)s - %(message)s"),
+            datefmt="%y-%m-%d %H:%M:%S",
+            stream=sys.stdout,
+        )    
+        logging.getLogger("urllib3").setLevel(logging.DEBUG)
+    else:
+        logging.basicConfig(
```

Comment:
I think this was originally set up so that running via terminal provides no logs, unless the verbose flag is set. So I don't think the `else` is required.

Not trying to be picky! But Rob set this up originally & I think it's quite common for CLI tools to have minimal logging unless verbose is specified 😄

Reply:
Got it. Wil update now.